soc/riscv/riscv-privilege/common/CMakeLists.txt

```diff
@@ -16,3 +16,9 @@ zephyr_sources(
 	idle.c
 )
 endif()
+
```

Comment:
strange fork. in latest develop i see
```
zephyr_sources(
	idle.c
)
```
https://github.com/telink-semi/zephyr/blob/develop/soc/riscv/riscv-privilege/telink_b9x/CMakeLists.txt
Just ensure that you are forked from latest develop. It was changed i thing month ago. Merging old state we introduce some degradation.